### DISCUSSION TITLE
Is it possible to adjust the crop by dragging the crop border?

### DISCUSSION THREAD

**Discussion by fiskpatte** [ValentinH/react-easy-crop#496] [General]
Hi, my question is if it's possible to adjust the crop by dragging it's border. So for example, if I left click the bottom crop border and drag down, then I want the crop area to expand in that direction.

**Comment by ValentinH** [comment]
No this is not possible and I actually created this library because I didn't want to crop by dragging borders.
If I were to implement such a UI, I would look into using https://github.com/DominicTobias/react-image-crop.
